<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpotViewer</class>
 <widget class="QDialog" name="SpotViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Монитор спотов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="filterButton">
     <property name="text">
      <string>Показать фильтры</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Фильтры</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Диапазон: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bandCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Модуляция: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeCombo">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Континент: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="contCombo">
          <item>
           <property name="text">
            <string>Все</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Африка</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Азия</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Европа</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Океания</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Южная Америка</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Северная Америка</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Страна: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="countryCombo">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Поиск DX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
